contents/handbook/growth/developer-relations.mdx

```diff
@@ -0,0 +1,232 @@
+
+However, there are metrics we can and should keep track of. Here are some examples:
+
+
+|       Area        |                                        Example Metrics                                        |             Is this currently tracked?             |
+| :---------------: | :-------------------------------------------------------------------------------------------: | :------------------------------------------------: |
+|       Docs        | "Was this page helpful" survey results, aggregate usage metrics (search, autofill, dark mode) |                        Yes                         |
+| Demos/Onboardings |                                     Conversion, Retention                                     | Retention is tracked, conversion is tracked poorly |
+|     Community     |   [Love](https://github.com/orbit-love/orbit-model), aggregate contribution metrics, stars    |           Yes, but could be done better            |
+|      Plugins      |                                         Plugin usage                                          |                        Yes                         |
+|     Tutorials     |                 Aggregate metrics, conversion (e.g. to discovered learnings)                  |                      Somewhat                      |
+
+### Potential areas to hire into
+
+#### DX
+
+As we grow, there are some clear areas we can hire into to decrease the breadth and increase the depth of the DX role:
+
+##### Demos and support
+
+While the majority of PostHog demos include a developer on the call (meaning we need a developer from our side too), most of them also end up as a Sales call or end up needing a follow-up Sales call.
+
+We can streamline this process by having someone who can do both run it. 
+
+**Example role:** Sales Engineer
+
+By removing this area of responsibility from DX, DX can then move more into user feedback and follow-up/setup calls rather than being the first touchpoint.
+
+##### Writing
+
+A dedicated copywriter will be able to do a better job of writing more top-of-funnel technical content aimed at acquiring new users.
+
+Additionally, given our short release cycles, the multiple facets of our product (Cloud, self-hosted, libraries, API, plugins, etc.), as well as our ambition for top-level design and feel for our Docs (token autofill, dark mode, search, etc.), getting our Docs to be world-class (as well as keeping them there) is definitely not a job for one single person - and potentially the job for a small team.
+
+#### DevRel as a whole
+
+On a more long-term view, we would also potentially benefit from having the following roles:
+
+- Developer Advocates/Evangelists/Educators: Acquistion-focused, external-facing, would take over some responsibilities from Marketing
+- Engineers for our libraries: Engineers who are also able to contribute to product but who would own certain key libraries (especially mobile libraries)
```

Comment:
I strongly suspect we'll have a small team focussed on our libraries and should expand our efforts here

We have already decided to hire a mobile developer at least as a contractor initially to make sure our mobile libraries get updated and work more nicely